packages/cli/src/build/index.js

```diff
@@ -100,10 +101,10 @@ async function build({flags}) {
   // Create demo.html files
   await Promise.all(patterns.map(async pattern => {
     const component = getComponent(bundles, pattern);
-    const context = getContext(pattern);
+    const context = getPatternContext(pattern);
```

Comment:
Would be great if we could await those, too.

Reply:
Changed in new commit.